### PR TITLE
* tldr.el: alter some strings and docstrings.

### PR DESCRIPTION
**Pull request by ghost**
The Emacs manual recommends that documentation strings are complete
sentences, beginning with a capital letter and ending with a full
stop, so I have made sure that all of them fit this.

I also reworded some of the sentences so that they sound more natural.
Nothing that I did changed the behaviour of the package, it was all
simply quality of life.

### CODE REVIEW

**Comment by arunisaac** [comment]
I think these changes are in the correct direction. +1 to including them in tldr.el.

Thanks! :-)

**Comment by ghost** [comment]
Thanks.  I'm not sure why it never got merged, but it appears to have a few conflicts now.  At some point I might go through and change them, but I haven't used tldr in a while now so I may not.

--
Kind regards,
Noah (distinctly.pink)

**Comment by kuanyui** [comment]
Ah... Sorry... Maybe because the old notification system of Github is too awful, as long as I missed any notification once then it will be forgotten forever......

**Comment by ghost** [comment]
No worries, it is a bit sucky :)

Well, I have resolved the conflict if you still want to merge my changes.  As I say, it doesn't affect any behaviour, just makes the docstrings sound a bit more fluid.

-- 
Noah (distinctly.pink)

**Comment by kuanyui** [comment]
Oops, still conflict
image

**Comment by ghost** [comment]
I'm not really sure what to do then, as on my end github is telling me
that this branch has no conflicts with the base branch.

Sorry :(

-- 
Noah (distinctly.pink)
